examples/complete-kubernetes-addons/main.tf

```diff
@@ -222,66 +176,47 @@ module "eks_blueprints_kubernetes_addons" {
 
   enable_amazon_eks_aws_ebs_csi_driver = true
 
-  # Prometheus and Amazon Managed Prometheus integration
-  enable_prometheus                    = true
-  enable_amazon_prometheus             = true
-  amazon_prometheus_workspace_endpoint = module.eks_blueprints.amazon_prometheus_workspace_endpoint
-
-  enable_aws_for_fluentbit = true
-  aws_for_fluentbit_helm_config = {
```

Comment:
do we have this in a different example?

Reply:
This wasn't intended - corrected in https://github.com/aws-ia/terraform-aws-eks-blueprints/pull/551/commits/3335bd466858a2527a90dad8256a0597a0a56262